test/integration/recover_remote_nse_nsm_heal_test.go

```diff
@@ -0,0 +1,136 @@
+// +build recover,!azure
```

Comment:
That's fine, but we really have to pursue fixing these later. Having a problem in one cloud probably means that is hidden in the others.

Reply:
Yep. There is definitely a problem that simply has a greater chance to reproduce on Azure.  I've created #961 to track that.